### PR TITLE
Update api-doc to say async functions are implemented

### PR DESCRIPTION
**Pull request by elliott-beach**
Signed-off-by: Elliott Beach <elliott2.71828@gmail.com>

## Motivation and Context
Async functions were implemented in #131, swagger doc says WIP

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] docs

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by elliott-beach** [comment]
working on gpg sign

File: api-docs/swagger.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ paths:
           description: Internal error with swarm or request JSON invalid
   /async-function/{functionName}:
     post:
-      summary: 'Invoke a function asynchronously in OpenFaaS - coming in PR 131'
+      summary: 'Invoke a function asynchronously in OpenFaaS'
```
Comment (johnmccabe) [inline api-docs/swagger.yml]: Do we need to call out the fact that this is only true if the stack is deployed in async mode - not the case by default right?

Comment (alexellis) [reply api-docs/swagger.yml]: Yes we do - but I think I'll merge because this is less wrong than saying about PR 131.

**Comment by alexellis** [comment]
@elliott-beach good catch. Do you think you can find a way to make a comment that this endpoint is optional as per John's comment?

**Comment by elliott-beach** [comment]
will do

**Pull request merged**